apps/meteor/tests/e2e/omnichannel-departaments.spec.ts

```diff
@@ -32,6 +34,16 @@ test.describe.serial('omnichannel-departments', () => {
 		await expect(poOmnichannelDepartments.firstRowInTable).toBeVisible();
 	});
 
+	test.skip('expect to not be possible adding a second department ', async () => {
```

Comment:
Do we need to put a skip at the top level as well when you already have a skip present on the next line? Won't that skip the tests also on EE?

Reply:
Yep, this test is for CE edition only, I'm just skipping entirely for now. We'll have to come back and improve these tests later on

Reply:
Oh, sorry, I meant, that adding a skip at line 37 might also skip this test for CE? Is that the intended behaviour? 
If you only wish to run this test on CE, then you may remove the skip over here on line 37, since there is already a check on the next line.